app/controllers/profile.js

```diff
@@ -10,16 +10,20 @@ export default Controller.extend({
   followUser: task(function*(profile) {
     this.toggleProperty('waitingForFollowing');
 
-    yield profile.follow().finally(() => {
+    try {
+      yield profile.follow();
+    } finally {
       this.toggleProperty('waitingForFollowing');
-    });
+    }
```

Comment:
Hmm looking at this again why are we even doing this?
Why not just have a computed property:
```js
waitingForFollowing: computed("followUser.isRunning", "unFollowUser.isRunning", {
  get() {
    return this.get("followUser.isRunning") || this.get("unFollowUser.isRunning");
  }
})
```

Does this make more sense?

Reply:
Yes, but I was just apply best patterns not really changing the logic. I can fix if you want.

Reply:
Perhaps it would make sense to use the `or` computer macro instead of `computed`?